releases.go

```diff
@@ -146,13 +146,13 @@ func (s *releasesService) Rollback(ctx context.Context, db *gorm.DB, opts Rollba
 		Slug:        r.Slug,
 		Formation:   r.Formation,
 		Description: desc,
-	})
+	}, nil)
 }
 
 // Release submits a release to the scheduler.
-func (s *releasesService) Release(ctx context.Context, release *Release) error {
+func (s *releasesService) Release(ctx context.Context, release *Release, ss scheduler.StatusStream) error {
 	a := newSchedulerApp(release)
-	return s.Scheduler.Submit(ctx, a)
+	return s.Scheduler.Submit(ctx, a, ss)
```

Comment:
If `ss` is nil, maybe want to create a null implementation and pass that through, so implementations of scheduler.Scheduler don't need to check for nil.